app/helpers/validate/throwMissingParameter.spec.js

```diff
@@ -0,0 +1,48 @@
+const assert = require("assert")
+const throwMissingParameter = require("./throwMissingParameter");
+
+describe("helpers/validate/throwMissingParameter", () => {
+    describe("fields should match object keys", () => {
+        it("matching keys", () => {
+            const object = {
+                keyone: "value",
+                keytwo: "another value"
+            }
+
+            const fields = ["keyone", "keytwo"]
+
+            throwMissingParameter(fields, object)
+        })
+        it("number of keys in object is greater than fields size", () => {
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 4 spaces but found 8.](https://app.codacy.com/gh/ufabc-next/ufabc-next-server/pullRequest?prid=8555381)
```suggestion
    it("number of keys in object is greater than fields size", () => {
```